src/web/components/Item/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Item({ content }) {
```

Comment:
Why not to use a children here instead ?

Reply:
I updated the component to use this way